src/main/java/winstone/Launcher.java

```diff
@@ -347,11 +346,10 @@ public static void main(String[] argv) throws IOException {
         if (System.getProperty("java.util.logging.config.file") == null) {
           for (Handler h : java.util.logging.Logger.getLogger("").getHandlers()) {
               if (h instanceof ConsoleHandler) {
-                  ((ConsoleHandler) h).setFormatter(new SupportLogFormatter());
+                  h.setFormatter(new SupportLogFormatter());
```

Comment:
I do not see how this change is related to the Jetty 10 upgrade. Suggest leaving unrelated changes to separate PRs for readability.

Reply:
you are right my bad. This cosmetic change in a middle of a documented breaking change definitely need his own PR, extended review and deserves his own entry in the release notes
here you go https://github.com/jenkinsci/winstone/pull/207

Reply:
you asked for a separate PR so automatically it will be an entry in the release notes.

Reply:
> automatically it will be an entry in the release notes

Not if one of [four exclusion labels](https://github.com/jenkinsci/.github/blob/4161800c96ca9cd294b5610b96995ee708a6ce9c/.github/release-drafter.yml#L52-L56) are used.

But that does not answer my question. You wrote that this _deserves_ an entry in the release notes (implying the exclusion labels should not be used). Why does this _deserve_ an entry in the release notes?

Reply:
> > automatically it will be an entry in the release notes
> 
> Not if one of [four exclusion labels](https://github.com/jenkinsci/.github/blob/4161800c96ca9cd294b5610b96995ee708a6ce9c/.github/release-drafter.yml#L52-L56) are used.
> 
> But that does not answer my question. You wrote that this _deserves_ an entry in the release notes (implying the exclusion labels should not be used). Why does this _deserve_ an entry in the release notes?

can we move forward please. I definitely have better to do in my life.